app/utils/kcidb.py

```diff
@@ -176,6 +179,21 @@ def push_tests(group_id, bq_options, db_options={}, db=None):
         'instance': group[models.BOARD_INSTANCE_KEY],
     }
 
+    files = {
+        'txt': group[models.BOOT_LOG_KEY],
+        'html': group[models.BOOT_LOG_HTML_KEY],
+        'lava_json': 'lava-{}.json'.format(group[models.DEVICE_TYPE_KEY]),
+    }
+
+    urls = dict()
+    output_files = []
+    for key in files:
```

Comment:
Looks a lot neater, but with a dict you can even do:
```python
for name, url in files.iteritems():
```
and then use `url` rather than `files[key]` 

Also, I'm not sure if the `urls` dictionary is needed as you should be able to just make a `url` string and use it to append an entry in `output_files`.  I don't think the `urls` dict is used anywhere else afterwards.

Reply:
OK, thanks.  Updated with those additional cleanups.